shm/src/store.cpp

```diff
@@ -88,12 +98,72 @@ void Store::nextDay() {
     }
 }
 
+void Store::generateRegularCargo() {
+    std::vector<std::shared_ptr<Cargo>> possibleRegularCargo{
+        std::make_shared<Alcohol>(Alcohol("Whisky", 0, 100, 47)),
+        std::make_shared<Alcohol>(Alcohol("Beer", 0, 12, 8)),
+        std::make_shared<Alcohol>(Alcohol("Wine", 0, 40, 47)),
+        std::make_shared<Fruit>(Fruit("Banana", 0, 5, 10)),
+        std::make_shared<Fruit>(Fruit("Orange", 0, 7, 7)),
+        std::make_shared<Fruit>(Fruit("Raspberry", 0, 30, 2)),
+        std::make_shared<Fruit>(Fruit("GoldenCucumber", 0, 70, 1)),
+    };
+
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    std::uniform_int_distribution<> initialAmount(0, 20);
+
+    for (const auto& cargo : possibleRegularCargo) {
+        *cargo += initialAmount(gen);
+        stock_.push_back(cargo);
+    }
+}
```

Comment:
I would put this inside a f-ction that takes one argument - possibleRegularCargo.

Reply:
Done.